rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -8448,6 +8624,23 @@ private static Stream<Arguments> getEstimatedFeesForNextPegOutEventArgsProvider(
         ).flatMap(Function.identity());
     }
 
+    private static Stream<Arguments> getEstimatedFeesForNextPegOutEventArgsProvider_postRSKIP305() {
```

Comment:
We might want to use better naming. There is a minor difference between this provider name and the one above, and it's not clear why we have 2

Reply:
Updated.